tests/impls/rank_relevancy/test_sorted_sampling.py

```diff
@@ -0,0 +1,53 @@
+import numpy
+from typing import Dict, List, Sequence, Iterable
+
+from smqtk_relevancy.interfaces.rank_relevancy import RankRelevancy
+from smqtk_relevancy.impls.rank_relevancy.sorted_sampling import (
+    RankRelevancyWithSortedFeedback,
+)
+from smqtk_descriptors import DescriptorElement
+
+
+def test_is_usable() -> None:
+    assert RankRelevancyWithSortedFeedback.is_usable()
+
+
+class DummyRankRelevancy(RankRelevancy):
+    def get_config(self) -> Dict:
+        return {}
+
+    def rank(self, pos: Iterable[DescriptorElement], neg: Iterable[DescriptorElement], pool: Sequence) -> List:
+        return [v[0] for v in pool]
+
+
+def make_sorted_ranker(n: int) -> "RankRelevancyWithSortedFeedback":
+    return RankRelevancyWithSortedFeedback(
+        DummyRankRelevancy(), n
+    )
+
+
+def test_parameter_n() -> None:
+    """
+    Check that the "n" parameter has the expected effect on feedback
+    request count
+    """
+    n = 10
+    mr = make_sorted_ranker(n)
+    for i in range(3, 31, 3):
+        pool = numpy.linspace(0, 1, i)[:, numpy.newaxis]
+        pool_uids = [object() for _ in range(i)]
+        scores, requests = mr.rank_with_feedback([], [], pool, pool_uids)
+        assert len(requests) == min(n, i)
+
+
+def test_pass_through() -> None:
+    """
+    Check that scores from the wrapped RankRelevancy are passed
+    through
+    """
+    mr = make_sorted_ranker(3)
+    pool = [[.3], [.1], [.45], [.29], [.03]]
+    expected = [.3, .1, .45, .29, .03]
+    uids = [object() for _ in pool]
+    scores, requests = mr.rank_with_feedback([], [], pool, uids)
+    assert list(scores) == expected
```

Comment:
We should have a test here that the feedback returned is in the correct descending-score order.

Reply:
Sorry, I didn't have a good place to put the comment, and I wasn't explicit about it, but I meant for that check to be it's own test function, not to be specifically in the `test_pass_through` unittest.

Reply:
Sounds good, added a new unittest.